jwst/ami/ami_analyze.py

```diff
@@ -78,7 +77,7 @@ def apply_LG_plus(input_model, filter_model, oversample, rotation):
     PIXELSCALE_r = pixelscale_as * arcsec2rad
     holeshape = 'hex'
     filt = "F430M"
-    rotsearch_d = np.arange(-3, 3.1, 1)
+    rotsearch_d = np.arange(rotsearch_parameters[0], rotsearch_parameters[1], rotsearch_parameters[2])
```

Comment:
It's odd that the defaults are such that the purpose of `[-3, 3.1, 1]` is to make an evenly-spaced interval between -3 and 3, spaced by 1.  But `[-3, 4, 1]` will produce the exact same thing.

```
In [25]: np.arange(-3, 3.1, 1)
Out[25]: array([-3., -2., -1.,  0.,  1.,  2.,  3.])

In [27]: np.arange(-3, 4., 1)
Out[27]: array([-3., -2., -1.,  0.,  1.,  2.,  3.])
```

Might be worth documenting that this is "start, stop, step" or similar in the spec comment.